gradle-guide-error-prone/src/main/java/com/palantir/gradle/guide/errorprone/taskexecution/GradleService.java

```diff
@@ -0,0 +1,59 @@
+/*
+ * (c) Copyright 2025 Palantir Technologies Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.palantir.gradle.guide.errorprone.taskexecution;
+
+import com.google.errorprone.VisitorState;
+import com.google.errorprone.suppliers.Supplier;
+import com.sun.tools.javac.code.Type;
+
+/**
+ * Represents a gradle service to be injected as part of an autofix.
+ */
+public class GradleService {
```

Comment:
I'd be tempted to make this an enum as there are only a certain number of `GradleService`s

Reply:
Good idea! I've made it an enum.